controllers/persistent-volume-claim/controller.go

```diff
@@ -26,17 +26,15 @@ const (
 
 // PersistentVolumeClaimReconciler reconciles a PersistentVolumeClaim object
 type PersistentVolumeClaimReconciler struct {
-	client    client.Client
-	apiReader client.Reader
-	recorder  record.EventRecorder
+	Client   client.Client
```

Comment:
Why the scope change? And more importantly, why are we removing the apiReader?

Reply:
apiReader was unused.
Scope Change as tests are in "persistent_volume_claim_test" package as per Go recommendations, and to inject fields without Getter/Setter I had to export them to be accesible in the test package